client/src/components/Sidebar.js

```diff
@@ -37,17 +50,26 @@ function Sidebar() {
 							About
 						</NavLink>
 					</li>
-					<li>
-					<NavLink
-							exact
-							to="/login"
-							activeStyle={{
-								color: "#fff",
-							}}
-						>
-							Login
-						</NavLink>
-					</li>
+					{authenticated === true && (
+						<li>
+							<a href="#" onClick={handleLogout}>
+								Logout
+							</a>
+						</li>
+					)}
+					{authenticated === false && (
+						<li>
+							<NavLink
+								exact
+								to="/login"
+								activeStyle={{
+									color: "#fff",
+								}}
+							>
+								Login
+							</NavLink>
+						</li>
+					)}
```

Comment:
sidebar:
show `Logout` if user is authenticated ( login )
show `Login` if user is not authenticated ( logout )